src/modules/layoutUiRules/rules.js

```diff
@@ -25,12 +26,14 @@ import {
     VIS_TYPE_PIVOT_TABLE,
     VIS_TYPE_SCATTER,
     VIS_TYPE_LINE_LIST,
+    VIS_TYPE_OUTLIER_TABLE,
 } from '../visTypes.js'
 
 const RULE_PROP_AVAILABLE_AXES = 'availableAxes',
     RULE_PROP_MAX_DIMS_PER_AXIS = 'maxNumberOfDimsPerAxis',
     RULE_PROP_MIN_DIMS_PER_AXIS = 'minNumberOfDimsPerAxis',
     RULE_PROP_MAX_ITEMS_PER_AXIS = 'maxNumberOfItemsPerAxis',
+    RULE_PROP_MAX_ITEMS_PER_DIMENSION = 'maxNumberOfItemsPerDimension',
```

Comment:
All other rules seems to shorten "dimensions" to "dims" so to follow that pattern this should be called `RULE_PROP_MAX_ITEMS_PER_DIM` then?

Reply:
Renamed.